src/screens/EnterPrivateKey/EnterPrivateKeyStyles.js

```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native'
+import {Header} from 'react-navigation'
+
+export default StyleSheet.create({
+  screenView: {
+    alignSelf: 'center',
+    width: '80%',
```

Comment:
This is incorrect. Width must be 100% and please implement it via `flex` and `flexDirection` with no 'width' in styles.